daprdocs/content/en/reference/components-reference/supported-state-stores/setup-mongodb.md

```diff
@@ -112,6 +112,10 @@ The username is `admin` by default.
 
 {{< /tabs >}}
 
+### TTLs and cleanups
+
+This state store supports [Time-To-Live (TTL)]({{< ref state-store-ttl.md >}}) for records stored with Dapr. When storing data using Dapr, you can set the `ttlInSeconds` metadata property to indicate when the data should be considered "expired".
```

Comment:
Can you also add `ttlInSeconds` to the example and the spec metadata table above?

Reply:
Mongodb doesn't currently support setting the ttl on the component spec so I left those out.